Makefile

```diff
@@ -299,6 +299,16 @@ quickstart: $(KUSTOMIZE) manifests #EXHELP Generate the installation release man
 
 ##@ Docs
 
+.PHONY: crd-ref-docs
+API_REFERENCE_FILENAME := operator-controller-api-reference.md
```

Comment:
Any strong thoughts on the filename? I prepended with `operator-controller` on the off-chance we put other APIs in here.

Reply:
`operator-controller-api-reference.md` sounds good to me.